core/tests/protractor_utils/workflow.js

```diff
@@ -118,17 +125,24 @@ var createExplorationAsAdmin = async function() {
 // This will only work if all changes have been saved and there are no
 // outstanding warnings; run from the editor.
 var publishExploration = async function() {
-  await element(by.css('.protractor-test-publish-exploration')).isDisplayed();
-  await element(by.css('.protractor-test-publish-exploration')).click();
+  var testPublishExplorationButton = element(
+    by.css('.protractor-test-publish-exploration'));
+  await testPublishExplorationButton.isDisplayed();
+  await action.click(
+    'testPublishExplorationButton', testPublishExplorationButton);
+
   var prePublicationButtonElem = element(by.css(
     '.protractor-test-confirm-pre-publication'));
   await prePublicationButtonElem.isPresent();
```

Comment:
Can you remove this too? Same reason as above. This way you don't need to add a `waitFor` statement above this line

Reply:
Hi, @U8NWXD please I don't really understand which lines I'm to remove here

Reply:
What you've done looks great!